### PR TITLE
Create a landing page for Buzzfeed

### PR DESCRIPTION
**Pull request by garethwilliamsdesign**
### Trello card

[Trello 5453](https://trello.com/c/5cHvrHp4)

### Context

As part of the campaign launch, the content and partnerships team have planned a partnership with Buzzfeed. We therefore need to create a landing page that Buzzfeed can direct users to who have engaged with the content on Buzzfeed and who want to find out more.

### Changes proposed in this pull request

Creation of a landing page for Buzzfeed.

### Guidance to review

Ensure the Buzzfeed landing page aligns with the Word document created by Maxine.

### CODE REVIEW

File: app/views/content/landing/what-teaching-taught-me/_collage.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="grey col">
+    <section class="col col-720 col-space-0 col-center associated-logos">
+      <div class="inset">
+        <p><strong>
+        In partnership with
+        </strong></p>
+      </div>
+      <div class="associated-logos-holder">
+        <%= image_pack_tag("static/images/content/grow/buzzfeed-red-logo-text.png", alt: "Buzzfeed logo") %>
```
Comment (gemmadallmandfe) [inline app/views/content/landing/what-teaching-taught-me/_collage.html.erb:10]: This looks a bit low-res and also squashed horizontally @garethwilliamsdesign ?

I also think we should check with the partnerships team whether we need the DfE logo - I suspect we don't (it wasn't in their draft doc) which might make things easier?

File: app/views/content/landing/what-teaching-taught-me/_promo.html.erb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section( classes: %w[ml-background]) %>
+  <% promo.right_section(
+    heading: "Find out more about getting into teaching"
+  ) do %>
+      <p>Find out more about funding your training, as well as top tips on making a successful application.</p>
```
[the remark this answers is not in the document]
Comment (MaxineCl) [reply app/views/content/landing/what-teaching-taught-me/_promo.html.erb:6]: I sent you a message about it on Slack. I couldn't change it yet as the page wasn't ready!

File: app/views/content/landing/what-teaching-taught-me/_collage.html.erb (hunk 1 above)
Comment (garethwilliamsdesign) [reply app/views/content/landing/what-teaching-taught-me/_collage.html.erb:10]: Yes, as I mentioned on Slack last night, the BuzzFeed logo looked great on my machine, but went all squiffy on GitHub. I'll fix this issue once it's been decided how many logos we want.

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me

**Pull request merged**